src/librustc_typeck/check/method/probe.rs

```diff
@@ -732,6 +734,49 @@ impl<'a, 'gcx, 'tcx> ProbeContext<'a, 'gcx, 'tcx> {
         Ok(())
     }
 
+    fn assemble_builtin_candidates(&mut self,
```

Comment:
This is going to be unnecessary after #43880 lands. I suppose the second PR to land will fix the rebase.